src/main/java/com/hydratereminder/dictionary/HydrateBreakMessageDictionary.java

```diff
@@ -70,6 +94,11 @@ public static String getRandomHydrateBreakMessageForPersonality(HydrateReminderP
                 final int randomNumber = randomGenerator.nextInt(HYDRATE_BREAK_FUNNY_TEXT_LIST.size());
                 breakMessage = HYDRATE_BREAK_FUNNY_TEXT_LIST.get(randomNumber);
                 break;
+            case CARING:
+                final SecureRandom randomGenerator = new SecureRandom();
+                final int randomNumber = randomGenerator.nextInt(HYDRATE_BREAK_CARING_TEXT_LIST.size());
```

Comment:
Hi Alex. Would you mind extracting the logic responsible for number randomization into a separate method?